Source/ProjectFirma.Web/Views/Project/DetailViewData.cs

```diff
@@ -139,7 +139,8 @@ public class DetailViewData : ProjectViewData
         public string ProjectInteractionEventsGridDataUrl { get;}
 
         public List<ProjectAgreementByGrantAllocation> ProjectAgreementByGrantAllocations { get; }

```

Comment:
remove this one too

Reply:
Done; removed declaration and assignment.